packages/lib/src/form-fields/file-list/file-list.wc.svelte

```diff
@@ -287,77 +314,138 @@
+                  /></svg
+                >
+              {/if}
+            </div>
+            <div class="file-name">
+              {file.name}
+            </div>
+            <div class="file-info">
+              <span>
+                {file.size || ''}
+              </span>
+              <span style="color: rgb(26 151 26)">
+                {file.saved ? 'saved' : ''}
+              </span>
+            </div>
           </div>
-          <div class="file-name">
-            {file.name}
-          </div>
-          <div class="file-info">
-            <span>
-              {file.size || ''}
-            </span>
-            <span style="color: rgb(26 151 26)">
-              {file.saved ? 'saved' : ''}
-            </span>
-          </div>
-        </div>
+        {:else}
+          <Cropper
+            maxImgHeight="85%"
+            maxImgWidth="85%"
+            maxContainerHeight="300px"
+            maxContainerWidth="85%"
+            src={file.src}
+            alt={file.name}
+            on:croppedImage={(e) => handleCrop(e)}
+            on:exitCropper={() => (beingCropped = null)}
+          />
+        {/if}
       {/each}
     </div>
-    <button type="button" class="add-more" on:click|preventDefault={() => browseFilesEl.click()}>
-      <svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 448 512"
-        ><path
-          d="M256 80c0-17.7-14.3-32-32-32s-32 14.3-32 32V224H48c-17.7 0-32 14.3-32 32s14.3 32 32 32H192V432c0 17.7 14.3 32 32 32s32-14.3 32-32V288H400c17.7 0 32-14.3 32-32s-14.3-32-32-32H256V80z"
-        /></svg
-      >
-    </button>
+    {#if beingCropped == null}
+      <button type="button" class="add-more" on:click|preventDefault={() => browseFilesEl.click()}>
+        <svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 448 512"
+          ><path
+            d="M256 80c0-17.7-14.3-32-32-32s-32 14.3-32 32V224H48c-17.7 0-32 14.3-32 32s14.3 32 32 32H192V432c0 17.7 14.3 32 32 32s32-14.3 32-32V288H400c17.7 0 32-14.3 32-32s-14.3-32-32-32H256V80z"
+          /></svg
+        >
+      </button>
+    {/if}
+  {:else}
+    {#each internalFiles as file, index}
+      {#if index === mandatoryCroped}
+        {#if file.type === 'image'}
+          <div class="cropperContainer">
+            {#if file.src}
+              {#if file.type === 'image'}
+                <Cropper
+                  src={file.src}
+                  alt={file.name}
+                  mandatory={true}
+                  on:croppedImage={(e) => handleMandatoryCrop(e)}
+                />
+              {/if}
+            {/if}
+          </div>
+        {:else}
+          <img
+            on:load={() => mandatoryCroped++}
+            src={'https://upload.wikimedia.org/wikipedia/commons/thumb/0/0f/Blank_Square.svg/525px-Blank_Square.svg.png'}
+            alt=" "
+          />
```

Comment:
This isn't a good way to load an image. It shouldn't be loaded from an external server.